tests/test_indexing.py

```diff
@@ -1,7 +1,7 @@
 # Number of entries in CDXJ == number of response records in WARC
 
 import pytest
-import testUtil as ipwbTest
+from . import testUtil as ipwbTest
```

Comment:
This change in the import style seems unrelated to this particular PR as it is more about Py3 support. If this is a dependency, then these changes should be made independently and merged prior to this PR.  Consider this comment for the rest of the changes in test files.

Reply:
Yeah, I did not originally intend of your code review of this, but because I used f-strings above, which are a Py3 feature, and the tests are still in Py2, we could not have the original changes in this PR without updating the tests to also be Py3.

Reply:
I am OK with Py3 changes and understand the dependency situation here, but my primary concern was the title is not reflective of changes. That's why I suggested that we make these Py3 changes in a separate PR, merge it in this branch, and then test this branch. However, if that is too much work, we can go ahead with this one as it is.

Reply:
@ibnesayeed I had started to do this...3 months ago in the (issue-51-tests)[https://github.com/oduwsdl/ipwb/tree/issue-51-tests] branch. The effort was never completed and I ended up redoing some of that here with additional fixes for the test.

I have updated the title of the PR to better reflect the changes. 😃